hw/bsp/mimxrt1010_evk/board.mk

```diff
@@ -1,2 +1,3 @@
-CHIP_FAMILY = mimxrt10xx
-CHIP_VARIANT = MIMXRT1011DAE5A
+TUF2_CHIP_FAMILY = mimxrt10xx
+TUF2_CHIP_MEMBER = mimxrt101x
```

Comment:
I'm not sure there's any benefit from having this one extra level `TUF2_CHIP_MEMBER` of specification for chip?

Reply:
Unfortunately there is because NXP, bless 'em, decided to pull an ST and make very similar numbered chips subtly different (specifically in pin numbering etc.) The extra level means we can keep the pinning_options, ld and mk files in separate directories per chip and the level above contain everything that is common across the entire family. This will become more important for ST, I fear.

Reply:
Sounds good, let's keep it then.